.github/workflows/tests.yaml

```
@@ -27,10 +27,9 @@ jobs:
           ${{ runner.os }}-go
     - name: checkout repository
       uses: actions/checkout@v2
-    - name: run integration tests
+    - name: run unit and integration tests
       run: make test.integration
```

```suggestion
      run: make test.unit test.integration
```

Given that you're removing a workflow that `make`s `test.unit`, don't we need to add `test.unit` here now that this step is supposed to run those?

---

We don't, the integration tests in this makefile run ALL tests, simply _including_ the integration tests (by way of go tag).

---

That's a misleading name for that target, then. Consider renaming to `test.all` alongside `test.integration` and `test.unit`

---

Sounds good, 09c27b175fd81ae78f7ebefa9bdbc36e5881b884 adds this :+1: